jobs/candlepinDockerImagesJob.groovy

```diff
@@ -0,0 +1,31 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+job("$baseFolder/candlepin-docker-images") {
+    previousNames('candlepin-docker-images')
+    description('Periodically builds and pushes docker images from master')
+    label('rhsm-jenkins-slave6')
```

Comment:
Does this line mean it'll only run on slave6? If so, is that what we want? I would argue the image building task should use any slave, unless the environment needs to be customized for it.